qcdb/intf_psi4/runner.py

```diff
@@ -2,216 +2,240 @@
 import copy
 import pprint
 pp = pprint.PrettyPrinter(width=120)
+from typing import Any, Dict, Optional
 import inspect
 
 import numpy as np
 import qcelemental as qcel
+from qcelemental.models import ResultInput
+
+import qcengine as qcng
+from qcengine.programs.util import PreservingDict
+from qcengine.programs.psi4 import Psi4Harness
+#from qcengine.programs.cfour.keywords import format_keywords, format_keyword
 
 from .. import __version__
 from .. import qcvars
-from ..driver.driver_helpers import print_variables
+#from ..driver.driver_helpers import print_variables
 from ..exceptions import *
+from ..util import provenance_stamp
 from ..molecule import Molecule
 from ..pdict import PreservingDict
-from .worker import psi4_subprocess
+#from .worker import psi4_subprocess
 from .botanist import muster_inherited_options
 
 
+#def run_psi4_old(name, molecule, options, **kwargs):
+#
+#    if kwargs['ptype'] not in ['energy', 'properties', 'gradient', 'hessian']:
+#        raise ValidationError("""run_psi4: ptype not regonized: {}""".format(ptype))
+#
+#    prov = {}
+#    prov['creator'] = 'QCDB'
+#    prov['version'] = __version__
+#    prov['routine'] = sys._getframe().f_code.co_name
+#
+#    jobrec = {}
+#    jobrec['error'] = ''
+#    jobrec['success'] = None
+#    jobrec['return_output'] = True
+#    jobrec['provenance'] = [prov]
+#    jobrec['molecule'] = molecule.to_dict(np_out=False)
+#    jobrec['method'] = name
+#    jobrec['driver'] = kwargs['ptype']
+#    jobrec['kwargs'] = kwargs
+#    jobrec['options'] = copy.deepcopy(options)
+#    jobrec['hooks'] = kwargs.get('hooks', {})
+#
+#    try:
+#        jobrec['molecule']
+#        jobrec['method']
+#    except KeyError as err:
+#        jobrec['error'] += repr(err) + 'Required fields missing from ({})'.format(jobrec.keys())
+#        return jobrec
+#
+#
+#    psi4rec = {}
+#    psi4rec['json'] = {}
+#
+#    opts = jobrec['options']
+#    # NOTE TODO very limited OPTIONS HANDSHAKE
+#    muster_inherited_options(opts)
+#
+#    psi4rec['json']['schema_name'] = 'qcschema_input'
+#    psi4rec['json']['schema_version'] = 1
+#    omem = opts.scroll['QCDB'].pop('MEMORY')
+#    psi4rec['json']['memory'] = omem.value
+#    psi4rec['json']['molecule'] = qcel.molparse.to_schema(jobrec['molecule'], dtype=2)
+#    psi4rec['json']['driver'] = jobrec['driver']
+#    mtd = jobrec['method']
+#    psi4rec['json']['model'] = {
+#        'method': mtd[3:] if mtd.startswith('p4-') else mtd,
+#        'basis': '(auto)',
+#    }
+#    psi4rec['json']['extras'] = {'wfn_qcvars_only': True}
```

Comment:
make sure this key is getting passed. it might fix the `DVEC`

Reply:
found the problem: the qcvar was being explicitly ignored in the old runner.

https://github.com/qcdb/qcdb/blob/452ba5d9592cc7fecbd81caaffaef90e5329b96d/qcdb/intf_psi4/runner.py#L118-L121